src/pages/profile/EditProfile.tsx

```diff
@@ -40,6 +35,7 @@ function EditProfile(): ReactElement {
     );
 
   const isFormLoading = isProfileDataLoading || !isProfileDataSuccess;
```

Comment:
you need to use (isProfileDataLoading || isCategoriesLoading), otherwise, if profile data is loaded before the categories, there will be an error on the screen (it can be seen when the page is reloaded for a split second)